newIDE/app/src/Utils/GDevelopServices/Game.js

```diff
@@ -40,20 +51,15 @@ export type Game = {
   acceptsBuildComments?: boolean,
   acceptsGameComments?: boolean,
   displayAdsOnGamePage?: boolean,
-};
+  cachedCurrentSlug?: CachedGameSlug,
```

Comment:
Is it really from a cache?
What does it contains? The last used slug or the default one if it's the first time?
Could it be a `suggestedSlug`?

Reply:
It is a "real" cached slug, in the sense that it's the slug currently assigned to the game (i.e: this is the value of the last "GameSlug" entity existing in the database).

A change in the backend will give a default slug for a new game (but this is unrelated).

But I think overall the meaning is correct:
- this is a cached value, i.e: a shortcut for something that exists in the backend as another entity (but you now don't need to make extra request to access it).
- this is a cached value, so the name implies it's read only and the backend needs to be cautious to update it if needed.